zerver/lib/email_mirror.py

```diff
@@ -201,7 +201,8 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
     if len(truncate_topic(topic)) < len(topic):
-        content = " ".join([topic, content])
+        content = "Subject: {}\n\n".format(topic) + content
```

Comment:
If you're using .format, isn't it more natural to have `` Subject: {}\n\n{}".format(topic, content) `` rather than mixing .format and string addition?